### PR TITLE
OCPBUGS#9197: Delete the nginx custom resource

### PR DESCRIPTION
**Pull request by xenolinux**
Version(s): 4.10+

Issue: https://issues.redhat.com/browse/OCPBUGS-9197

Link to docs preview: https://57345--docspreview.netlify.app/openshift-enterprise/latest/operators/operator_sdk/helm/osdk-helm-tutorial.html#osdk-create-cr_osdk-helm-tutorial

QE review:
- [x] QE has approved this change.

Additional information:

### CODE REVIEW

**Comment by Xia-Zhao-rh** [comment]
@emmajiafan  Would you please help to review it?

File: modules/osdk-create-cr.adoc [hunk 1]
```diff
@@ -200,6 +200,13 @@ memcached-sample                        5/5     5            5           3m
 ----
 endif::[]
 
+. Delete the `nginx` CR by running the following command:
++
+[source,terminal]
+----
+$ oc delete -f config/samples/demo_v1alpha1_nginx.yaml
```
Comment (Xia-Zhao-rh) [inline modules/osdk-create-cr.adoc:207]: Better to change to bellow

. Delete the CR:
+
[source,terminal,subs="attributes+"]
----
$ oc delete -f config/samples/{group}_v1_{app}.yaml
----

Comment (xenolinux) [reply modules/osdk-create-cr.adoc]: Addressed

File: modules/osdk-create-cr.adoc [hunk 2]
```diff
@@ -200,6 +200,13 @@ memcached-sample                        5/5     5            5           3m
 ----
 endif::[]
 
+. Delete the CR by running the following command:
```
Comment (Xia-Zhao-rh) [inline modules/osdk-create-cr.adoc:203]: Better to change to "Delete the CR:", no need to add "by running the following command"

Comment (xenolinux) [reply modules/osdk-create-cr.adoc:203]: @Xia-Zhao-rh As per the OS docs guidelines, it is better to mention "by running the following command" for clarity.

Comment (Xia-Zhao-rh) [reply modules/osdk-create-cr.adoc:203]: OK, I hava no other comments. lgtm

**Review by snarayan-redhat — COMMENTED** [review]
Looks good!

**Pull request merged**